soul-web/src/main/java/org/dromara/soul/web/configuration/SoulConfiguration.java

```diff
@@ -161,6 +162,17 @@ public WebFilter fileSizeFilter(final SoulConfig soulConfig) {
         return new FileSizeFilter(soulConfig.getFileMaxSize());
     }
 
+    /**
+     * Rule out the url Filter.
+     *
+     * @return the web filter
+     */
+    @Bean
+    @Order(-5)
+    public WebFilter excludeFilter() {
```

Comment:
The configuration is not enabled by default

Reply:
down.